libraries/cloudharness-common/cloudharness/utils/server.py

```diff
@@ -110,6 +110,7 @@ def handle_exception(e: Exception):
             except:
                 logging.error("Error checking sentry configuration", exc_info=True)
                 data['trace'] = traceback.format_exc()
+            logging.error(str(e), exc_info=True)
```

Comment:
I don't see why this is needed here. If we want to directly log the issue in Sentry then please use the Sentry sdk to create a Sentry message (f.e. use `capture_message` from the sentry_sdk)

Reply:
I don't think Sentry is a replacement of base server logging: Sentry is an optional and nice to have addition.